Sources/App/API/Model/Video.swift

```diff
@@ -14,6 +14,7 @@ final class Video: Content, Codable {
     var external: Bool?
     var createdAt: Date?
     var videoDate: Date?
+    var isUpcoming: Bool?
```

Comment:
can this be used as a computed property like twitterText?

Reply:
Actually I tried to use it that way. However, it was not possible. As far as what I experienced. That is because it twitterText example it was molded into the context. Thus it was being computed. However, when a var is called within the leaf it is not being computed as it should be :( So I tried to use context element as intermediary within the new commit. Let's discuss it when you are available. Thank you !